### PR TITLE
Add titleBarStyle for WindowOptions

### PR DESCRIPTION
**Pull request by liasica**

### CODE REVIEW

**Comment by asticode** [comment]
Thanks for the PR.

Could you put `TitleBarStyle` in alphabetical order in the struct as well as provide enums for the possible values such as :

```go
// Title bar styles
var (
    TitleBarStylesMyValue = PtrString("my value)
)
```

Cheers

**Comment by liasica** [comment]
Ok, like this ?
`// Title bar styles
var (
    TitleBarStyleDefault     = PtrStr("default")
    TitleBarStyleHidden      = PtrStr("hidden")
    TitleBarStyleHiddenInset = PtrStr("hidden-inset")
)`

**Comment by asticode** [comment]
Perfect. Only alphabetical order missing now and I'll merge the PR.

File: window.go [hunk 1]
```diff
@@ -67,6 +74,7 @@ type WindowOptions struct {
 	Width                  *int    `json:"width,omitempty"`
 	X                      *int    `json:"x,omitempty"`
 	Y                      *int    `json:"y,omitempty"`
+	TitleBarStyle 	       *string `json:"titleBarStyle,omitempty"`
```
Comment (asticode) [inline window.go]: Could you put it in alphabetical order in the struct?

**Comment by asticode** [comment]
I've commented the files changed, could you take a look?

File: window.go (hunk 1 above)
Comment (liasica) [inline window.go]: Done 👍 :)

**Comment by asticode** [comment]
LGTM

**Pull request merged**